docker/nginx/nginx.conf

```diff
@@ -128,7 +128,16 @@ http {
                 client_body_timeout         10m;
                 send_timeout                10m;
                 proxy_read_timeout          10m;
-                client_max_body_size        3G;
+                client_max_body_size        100G;
+                proxy_pass  http://${APP_HOST}:${APP_PORT};
+            }
+
+            location ~ ^/(datastore/file/add|datastore/file/add-interactive) {
+                keepalive_timeout           10m;
+                client_body_timeout         10m;
+                send_timeout                10m;
+                proxy_read_timeout          10m;
+                client_max_body_size        100G;
```

Comment:
From 1G and upper, we still get a 413 from the proxy.

Reply:
Fixed in https://github.com/dfir-iris/iris-web/pull/115/commits/aef41eb44c65bc082e2defafb6e527b628a66432.  
There was a limit set in Flask itself.